README.md

```diff
@@ -28,8 +28,7 @@ export PORT=... # The TCP port where the application will listen for HTTP traffi
 export LOG_LEVEL=... # The threshold level for logs to be written (e.g. DEBUG, INFO, WARN, or ERROR) (default: INFO)
 export MSA_ENTITY_ID=... # The SAML Entity Id that identifies the Relying Party's Matching Service Adapter
 export MSA_METADATA_URL=... # The URL to the Matching Service Adapter's SAML metadata.
-export HUB_METADATA_URL=... # The URL to the Verify Hub's SAML metadata. (default: https://www.signin.service.gov.uk/SAML2/metadata/federation)
-export HUB_SSO_LOCATION=... # The URL to send SAML AuthnRequests to the Verify Hub
+export HUB_ENVIRONMENT=... # The environment of the Verify Hub to run against - PRODUCTION, INTEGRATION, or COMPLIANCE_TOOL
```

Comment:
I wonder if `VERIFY_ENVIRONMENT` might be a better name. Not sure people outside this team actually understand the term `HUB` the way we do.